docs/configuring_the_eos_plugin.md

```diff
@@ -22,5 +25,9 @@ To get the EOS working, the plugin needs to know some specific things about your
     * [ClientSecret](https://dev.epicgames.com/docs/services/en-US/Glossary/index.html#C?term=OAuth%20ClientSecret)
     * [ClientID](https://dev.epicgames.com/docs/services/en-US/Glossary/index.html#C?term=OAuth%20ClientId)
     * EncryptionKey
-   
+
+    <br />
+
+    <img src="images/dev-portal-configuration-window.png" width="550" />
+    
```

Comment:
Not sure if these credentials should be filtered, even if users do have access to our config files.

Reply:
In previous iterations of this document, the same content was displayed, so this does not represent a change in what we make available to users via documentation.

Reply:
Fixed this.